contrib/crio-wipe/test.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
```

Comment:
Let's use bash everywhere.

Reply:
Also, we could use bats here similar to our crio suite.

Reply:
Done!